rest/route/service.go

```diff
@@ -119,7 +119,7 @@ func AttachHandler(app *gimlet.APIApp, opts HandlerOpts) {
 	app.AddRoute("/hosts/{task_id}/list").Version(2).Get().Wrap(checkTask).RouteHandler(makeHostListRouteManager(sc))
 	app.AddRoute("/hosts/{host_id}/attach").Version(2).Post().Wrap(checkUser).RouteHandler(makeAttachVolume(sc, env))
 	app.AddRoute("/hosts/{host_id}/detach").Version(2).Post().Wrap(checkUser).RouteHandler(makeDetachVolume(sc, env))
-	app.AddRoute("/hosts/{host_id}/provisioning_script").Version(2).Get().Wrap(checkHost).RouteHandler(makeHostProvisioningScriptGetHandler(sc))
+	app.AddRoute("/hosts/{host_id}/provisioning_options").Version(2).Get().Wrap(checkHost).RouteHandler(makeHostProvisioningOptionsGetHandler(sc))
```

Comment:
Will changing this make in-flight provisioning hosts fail right after the deploy? Maybe it's not a big deal either way.

Reply:
No because this isn't being used anywhere yet.